### ISSUE TITLE
Unable to set config for google ai studio gemini api 

### ISSUE THREAD

**Issue by mehnoorsiddiqui** [promptfoo/promptfoo#451]
![image](https://github.com/promptfoo/promptfoo/assets/57288492/a12a4dc2-0758-4757-a546-b98365f4df63)
I am setting the config like below:

```
providers:
  - id: google:gemini-pro   
    config:
      temperature: 0
      topP: 1
```

**Comment by typpo** [comment]
Thanks for the catch.  I feel like Google has kept moving things around in their API.  #454 will fix support for `temperature` and `topP` and other generation config options.

**Comment by typpo** [comment]
This fix is published in 0.40.0
